Classes/Service/NodePolicyService.php

```diff
@@ -102,6 +103,22 @@ public function isNodeTreePrivilegeGranted(NodeInterface $node): bool
         );
     }
 
+    /**
+     * @param NodeInterface $node
+     * @return bool
+     */
+    public function canReadNode(NodeInterface $node): bool
+    {
+        if (!isset(self::getUsedPrivilegeClassNames($this->objectManager)[ReadNodePropertyPrivilege::class])) {
```

Comment:
The symbol `ReadNodePropertyPrivilege` was not imported and thus this is just a noop and returns always `true`